packages/react-core/src/Authenticator/hooks/useAuthenticator/types.ts

```diff
@@ -38,6 +38,7 @@ export interface UseAuthenticator extends AuthenticatorServiceFacade {
   getTotpSecretCode: () => Promise<string>;
 
   /** @deprecated For internal use only */
+  QRFields: { totpIssuer?: string; totpUsername?: string };
```

Comment:
I made this deprecated internal, because we eventually want to handle this differently.

Reply:
Should `fields` also be `@deprecated?

Reply:
OK, I'll update it!

Reply:
[d681c00](https://github.com/aws-amplify/amplify-ui/pull/3183/commits/d681c00fe7843e52b7b95834c9092c64c308781d)